### DISCUSSION TITLE
Is there a fold list prefab in MRTK3?

### DISCUSSION THREAD

**Discussion by acg78219** [microsoft/MixedRealityToolkit-Unity#11759] [Q&A]
I know there have a "VirtualizedScrollRectList" scene, but I also want to make a list which can fold or unfold, like when I click a list item, then show its son item. How can I do it?

**Comment by AMollis** [comment]
@acg78219, I'm not sure I understand what you mean by "unfold".   Are you wanting to have a single list item expand to a larger size, while the others remain smaller?  Having variable item sizes in a virtualized list is currently not supported.    

I suggest you create an MRTK3 feature request for this ask.   Please provide your scenario details the best you can.  It would also be helpful if you provided design mockups of your scenario.  

Thank you

**Reply by acg78219** [reply]
yep, forgive my poor english, just as you describe, maybe I should call it "collapsible list".  XD
